package.json

```diff
@@ -17,6 +17,7 @@
         "@types/highlight.js": "9.12.4",
         "@types/jest": "26.0.14",
         "@types/js-yaml": "3.12.5",
+        "@types/luxon": "^1.25.0",
```

Comment:
Pin

Reply:
It will get auto pinned on saturday, but okay